package.json

```
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "build:miniconda": "curl -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && bash ~/miniconda.sh -b -p $HOME/miniconda",
-    "build:pygmt": "conda env create -f environment.yml && source activate pygmt && conda install -c conda-forge -y gmt==6.1.1 && make install",
+    "build:pygmt": "conda env create -f environment.yml && source activate pygmt && make install",
```

```suggestion
    "build:pygmt": "conda env create -f environment.yml && conda activate pygmt && make install",
```

Try if `conda activate` works now.

Was also thinking of switching to use [mamba](https://github.com/mamba-org/mamba) instead of conda to speed if it can speed up our CI as it can does parallel download of the libraries.

---

It doesn't work. I think it means we need to run `conda init bash` before `conda activate pygmt`, right?
```
22:09:19.563 | #
-- | --
22:09:19.563 | # To activate this environment, use
22:09:19.563 | #
22:09:19.563 | #     $ conda activate pygmt
22:09:19.563 | #
22:09:19.563 | # To deactivate an active environment, use
22:09:19.563 | #
22:09:19.563 | #     $ conda deactivate
22:09:22.134 | CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
22:09:22.134 | To initialize your shell, run
22:09:22.135 | $ conda init <SHELL_NAME>
22:09:22.135 | Currently supported shells are:
22:09:22.135 | - bash
22:09:22.135 | - fish
22:09:22.135 | - tcsh
22:09:22.135 | - xonsh
22:09:22.135 | - zsh
22:09:22.135 | - powershell


```

---

Ok, then just stick with `source activate`